### PR TITLE
Fix ruby 1.8 style hashes.

### PR DESCRIPTION
**Pull request by jstraw**
Depends on #627 for CI fixes

### CODE REVIEW

**Comment by binford2k** [comment]
tests depend on #627 being resolved

**Comment by jstraw** [comment]
Per discussion in the hackathon channels, this has been force pushed back to just the rubocop fixes and updated description to note that it requires the CI fixes in #627 to merge first

**Comment by jordanbreen28** [comment]
@jstraw https://github.com/puppetlabs/puppetlabs-kubernetes/pull/627 was merged in, can you resolve the conflicts and the tests should re-kick?
Thanks!

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Style/HashSyntax:
-  UseHashRocketsWithSymbolValues: true
+#Style/HashSyntax:
```
Comment (jordanbreen28) [inline .rubocop_todo.yml:1]: can we remove this cop from the todo

Comment (jordanbreen28) [reply .rubocop_todo.yml]: @jstraw can we not remove this completely from the todo now you've corrected it? Correct me if im wrong.

**Comment by jstraw** [comment]
Changes were cherry picked in puppet 8 work.

**Pull request closed without merging**